### PR TITLE
Specify that scramble signatures *should* be used.

### PR DESCRIPTION
**Pull request by lgarron**
@Laura-O, could you review?

### CODE REVIEW

File: wca-regulations.md [hunk 1]
```diff
@@ -376,7 +376,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - A2c) After the scrambler starts scrambling the puzzle, the competitor must not see the puzzle until the inspection phase starts.
         - A2c1) The scrambler places a cover over the scrambled puzzle that makes it impossible for any competitors or spectators to see any part of the puzzle. The cover remains over the puzzle until the start of the attempt.
     - A2d) When taking a puzzle from the scrambler, the judge briefly inspects the puzzle to ensure thorough scrambling of the puzzle. The judge raises any concerns with the scrambler, who then conducts a detailed check.
-        - A2d1) The organization team may require scramblers to take additional measures to make sure that they have applied the correct scramble sequence (e.g. signing the score sheet to affirm that they applied the correctly-numbered scramble sequence and checked that it matches the image, or using an additional person as "scramble checker").
+        - A2d1) The organization team should require scramblers to sign the score sheet to affirm that they applied the correctly-numbered scramble sequence and checked that it matches the image.
```
Comment (AlbertoPdRF) [inline wca-regulations.md]: I think I would add an "(or initial)" here.

**Review by AlbertoPdRF — APPROVED** [review]
One small comment, but LGTM!

**Review by Laura-O — APPROVED** [review]
Agree with Alberto, the rest LGTM!

File: wca-regulations.md (hunk 1 above)
Comment (lgarron) [reply wca-regulations.md]: Thanks, done!

**Pull request merged**